### ISSUE TITLE
Resource order in client is random

### ISSUE THREAD

**Issue by Mierdin** [StackStorm/st2#3927]
Should sort alphabetically or something.

```
(virtualenv) vagrant@st2dev:~/st2$ st2 trigger list
+--------------------------------------+----------+--------------------------------------------------------------------------------+
| ref                                  | pack     | description                                                                    |
+--------------------------------------+----------+--------------------------------------------------------------------------------+
| core.st2.IntervalTimer               | core     | Triggers on specified intervals. e.g. every 30s, 1week etc.                    |
| core.st2.DateTimer                   | core     | Triggers exactly once when the current time matches the specified time. e.g.   |
|                                      |          | timezone:UTC date:2014-12-31 23:59:59.                                         |
| core.st2.CronTimer                   | core     | Triggers whenever current time matches the specified time constaints like a    |
|                                      |          | UNIX cron scheduler.                                                           |
| core.st2.generic.actiontrigger       | core     | Trigger encapsulating the completion of an action execution.                   |
| core.st2.generic.notifytrigger       | core     | Notification trigger.                                                          |
| core.st2.action.file_writen          | core     | Trigger encapsulating action file being written on disk.                       |
| core.st2.key_value_pair.create       | core     | Trigger encapsulating datastore item creation.                                 |
| core.st2.key_value_pair.update       | core     | Trigger encapsulating datastore set action.                                    |
| core.st2.key_value_pair.value_change | core     | Trigger encapsulating a change of datastore item value.                        |
| core.st2.key_value_pair.delete       | core     | Trigger encapsulating datastore item deletion.                                 |
| core.st2.sensor.process_spawn        | core     | Trigger indicating sensor process is started up.                               |
| core.st2.sensor.process_exit         | core     | Trigger indicating sensor process is stopped.                                  |
| examples.sample_trigger              | examples | Sample trigger                                                                 |
| testpack.event                       | testpack | An example trigger.                                                            |
| examples.polling_event               | examples | An example trigger.                                                            |
| examples.event                       | examples | An example trigger.                                                            |
| core.st2.webhook                     | core     | Trigger type for registering webhooks that can consume arbitrary payload.      |
| core.st2.generic.inquiry             | core     | Trigger indicating a new "inquiry" has entered "pending" status                |
+--------------------------------------+----------+--------------------------------------------------------------------------------+
```

**Comment by LindsayHill** [comment]
https://github.com/StackStorm/st2/pull/4348 addresses this

**Comment by LindsayHill** [comment]
Fixed by #4348
